src/data/fr/index.ts

```diff
@@ -406,7 +407,7 @@ export const loggedInData = {
       directLink: "Lien direct vers ce contenu"
     },
     notifications: {
-      hide: "Masquer les notifications pour ce contenu.",
+      hide: 'Deactivate new notifications for this content.',
```

Comment:
@elbotho It looks like we are losing some translations?

Reply:
If de is complete this is fine. I actually changed the source strings here so the i18n crew has to update at some point.